docs/api.md

```diff
@@ -354,6 +393,51 @@ const FocusableNode = () => (
 
 In the example above, the third node will get the default focus when mounting our page.
 
+# SpatialNavigationDeviceTypeProvider
+
+This context allows to know what devices is used to control the focus. It handles two type of device :
+
+- `remoteKeys` which represents the TV remote's buttons
+- `remotePointer` which represents the TV remote pointer (available on certain TVs, like LG and their Magic Remote for example). It is also triggered by a mouse on the web.
+
+## How does it work
+
+By default, the context provides the value `remoteKeys` as it is the most widely use type of device.
+Every time an input is made on a device which is different from the one specified in the context, its value changes to represent the most recent device used by the user.
+
+## Usage
+
+```jsx
+function App(): JSX.Element {
+  return (
+    <NavigationContainer>
+      <ThemeProvider theme={theme}>
+        <SpatialNavigationDeviceTypeProvider>
+          <GoBackConfiguration />
+
+          <Container width={width} height={height}>
+            <Stack.Navigator
+              screenOptions={{
+                headerShown: false,
+                contentStyle: {
+                  backgroundColor: theme.colors.background.main,
+                },
+              }}
+              initialRouteName="TabNavigator"
+            >
+              <Stack.Screen name="TabNavigator" component={TabNavigator} />
+              <Stack.Screen name="ProgramDetail" component={ProgramDetail} />
+            </Stack.Navigator>
+          </Container>
+        </SpatialNavigationDeviceTypeProvider>
+      </ThemeProvider>
+    </NavigationContainer>
+  );
+}
+```
```

Comment:
You can simplify this I think 🤔 

```suggestion
```tsx
function App(): JSX.Element {
  return (
    <SomeOtherProviders>
        <SpatialNavigationDeviceTypeProvider>
          <YourApp/>
        </SpatialNavigationDeviceTypeProvider>
    </SomeOtherProviders>
  );
}
```
```

Reply:
Done :)